lib/Doctrine/Common/Cache/FileCache.php

```diff
@@ -124,11 +112,13 @@ public function getExtension()
      */
     protected function getFilename($id)
     {
+        $hash = hash('sha256', $id);
```

Comment:
You can do this sort of assumption in your own technical domain, but not in a general purpose lib.